### PR TITLE
feat: remove numpy

### PR DESCRIPTION
**Pull request by cristianmtr**
Closes https://github.com/jina-ai/jina/issues/3110

AFTER APPROVAL / BEFORE MERGING

on Hubble

- remove Numpy
- remove Compounds with Numpy
- push new Compounds
- add secrets

### CODE REVIEW

File: jinahub/indexers/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ See below figure for how this would look like:
 
 ![](./.github/img/replicas.png)
 
-In the above case, the Storage could be the [PostgreSQL](storage/PostgreSQLStorage)-based Storage, while the Query Flow could be based on [NumpyPostgresSearcher](searcher/compound/NumpyPostgresSearcher).
+In the above case, the Storage could be the [PostgreSQL](storage/PostgreSQLStorage)-based Storage, while the Query Flow could be based on [NumpyPostgresSearcher](searcher/compound/FaissPostgresSearcher).
```
Comment (tadejsv) [inline jinahub/indexers/README.md:24]:
```suggestion
In the above case, the Storage could be the [PostgreSQL](storage/PostgreSQLStorage)-based Storage, while the Query Flow could be based on [FaissPostgresSearcher](searcher/compound/FaissPostgresSearcher).
```

**Comment by tadejsv** [comment]
Are you still having problems here with non-consistent results?

**Comment by cristianmtr** [comment]
> Are you still having problems here with non-consistent results?

Yes, that's why the check is only `>`

**Review by Yongxuanzhang — COMMENTED** [review]
If we remove the searcher, do we also want to remove it from bubble? @cristianmtr

**Comment by cristianmtr** [comment]
> If we remove the searcher, do we also want to remove it from bubble? @cristianmtr

Yes, good point. We should. Once this gets approved, I'll remove it, along with the compounds, and add these two new Compounds

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**